Automattic/jetpack
# Fix Contact Form inline block inserter

Fixes #26888

#### Changes proposed in this Pull Request:

Before these changes, if you hover the mouse between two Form child blocks, the Block Inserter doesn't show.
The Block Inserter only shows if there's some margin between the blocks.
The Form child blocks were adding some spacing utilizing a `border-bottom: 15px solid transparent` instead of using some ` margin-bottom` and this was preventing the Block Inserter from being triggered.

This PR changes the child blocks styles to use `margin` over the `border-bottom`.
It also includes a few more styling changes to make the Form layout more consistent across the Editor and the Live/Preview view.

![image](https://user-images.githubusercontent.com/7811225/195615976-abccff64-edfe-49ad-90fd-37728cecbd6c.png)

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Checkout this branch and insert a Form on a Page/Post
* Hover your mouse between two Form child blocks
* The Block Inserter should appear, allowing you to easily insert new fields between the blocks
